packages/amplify_api/test/graphql_helpers_test.dart

```diff
@@ -23,7 +23,7 @@ import 'package:flutter_test/flutter_test.dart';
 
 void main() {
   group('with ModelProvider', () {
-    final AmplifyAPI api = AmplifyAPI(modelProvider: ModelProvider.instance);
+    AmplifyAPI(modelProvider: ModelProvider.instance);
```

Comment:
Prefer to delete

Reply:
You want to delete this line completely? maybe I'm missing something, but this is needed to register the model provider in the plugin singleton, just the plugin as a value not needed after that (at least to test model helpers).